Sources/CoreFoundation/CFRunArray.c

```diff
@@ -200,6 +200,14 @@ CFRunArrayRef CFRunArrayCreate(CFAllocatorRef allocator) {
     return array;
 }
 
+CFRunArrayRef CFRunArrayRetain(CFRunArrayRef array) {
```

Comment:
Is `CFRunArray` not a regular CF type? (`CFRetain`, `CFRelease` should work)

Reply:
When I try to use CFRetain, I get the following error:
> 'CFRetain' is unavailable: Core Foundation objects are automatically memory managed

Reply:
So if it’s automatically memory managed, where is the leak coming from? We don’t have to do this with other CF types.